docs/pages/admin-guide.mdx

```diff
@@ -411,80 +381,66 @@ https://<proxy>:3080/web/newuser/xxxxxxxxxxxx
 
 ```bash
-$ tctl users ls
+tctl users ls
 
+# Output
 User           Allowed Logins
 ----           --------------
 admin          admin,root
 ross           ross
 joe            joe,root
 ```
+{/* Convert to new UI component https://github.com/gravitational/next/issues/275 */}
 
-Joe would then use the `tsh` client tool to log in to member node "luna" via
-bastion "work" *as root*:
+Joe would then use the `tsh` client tool to log in to member node "luna" via bastion "work" *as root*:
 
 ```bash
-$ tsh --proxy=work --user=joe root@luna
+tsh --proxy=work --user=joe root@luna
 ```
 
 To delete this user:
 
 ```bash
-$ tctl users rm joe
+tctl users rm joe
 ```
 
-## Editing Users
+## Editing users
 
-Users entries can be manipulated using the generic [resource
-commands](#resources) via [`tctl`](cli-docs.mdx#tctl) . For example, to see the
-full list of user records, an administrator can execute:
+Users entries can be manipulated using the generic [resource commands](#resources) via [`tctl`](cli-docs.mdx#tctl) . For example, to see the #full list of user records, an administrator can execute:
 
 ```bash
-$ tctl get users
+tctl get users
 ```
 
 To edit the user "joe":
 
 ```bash
-# dump the user definition into a file:
-$ tctl get user/joe > joe.yaml
+# Dump the user definition into a file:
+tctl get user/joe > joe.yaml
 # ... edit the contents of joe.yaml
 
-# update the user record:
-$ tctl create -f joe.yaml
+# Update the user record:
+tctl create -f joe.yaml
 ```
 
-Some fields in the user record are reserved for internal use. Some of them will
-be finalized and documented in the future versions. Fields like `is_locked` or
-`traits/logins` can be used starting in version 2.3
+Some fields in the user record are reserved for internal use. We plan on finalizing and documenting these in future versions. Fields like `is_locked` or `traits/logins` can be used starting in version 2.3
 
-## Adding Nodes to the Cluster
+## Adding Nodes to the cluster
 
-Teleport is a "clustered" system, meaning it only allows access to nodes
-(servers) that had been previously granted cluster membership.
+Teleport is a "clustered" system, meaning it only allows access to nodes (servers) that had been previously granted cluster membership.
 
-A cluster membership means that a node receives its own host certificate signed
-by the cluster's auth server. To receive a host certificate upon joining a
-cluster, a new Teleport host must present an "invite token". An invite token
-also defines which role a new host can assume within a cluster: `auth` , `proxy`
-or `node` .
+A cluster membership means that a node receives its own host certificate signed by the cluster's auth server. To receive a host certificate upon joining a
+cluster, a new Teleport host must present an "invite token". An invite token also defines which role a new host can assume within a cluster: `auth`, `proxy`, or `node`.
```

Comment:
We should add a note about app, kube and db invite tokens.

Reply:
Great catch! This needs to be updated elsewhere too. I've created a ticket for this so we can be a bit more thorough: https://github.com/gravitational/teleport/issues/7147. I will also update this here.